_rules/line-height-not-important-78fd32.md

```diff
@@ -185,6 +189,61 @@ The [computed][] value of the `line-height` property of this `p` element is **no
 </p>
 ```
 
+#### Passed Example 11
+
+This `p` element only has one text node descendant, and it is [large enough][] because its parent is the `span` element, with a `line-height` of `24px`, more than `1.5` times the `font-size` of `16px`.
+
+```html
+<p style="font-size: 24px; line-height: 24px !important;  max-width: 200px;">
+	<span style="font-size: 16px;">
+		The toy brought back fond memories of being lost in the rain forest.
+	</span>
+</p>
+```
+
+#### Passed Example 12
+
+This `p` has three text nodes descendants. The first and third are [large enough][]. The second one is [empty](#78fd32:empty)
+
+```html
+<p style="font-size: 24px; line-height: 24px !important;  max-width: 200px;">
+	<span style="font-size: 16px;">
+		The toy brought back fond memories of being lost in the rain forest.
+	</span>
+	<span>&nbsp;</span>
+	<span style="font-size: 16px;">
+		The toy brought back fond memories of being lost in the rain forest.
+	</span>
+</p>
+```
+
+#### Passed Example 13
+
+Both this `p` and `span` elements have a single text node descendant, whose parent (the `span` element) is [large enough][].
+
+```html
+<p style="font-size: 16px; line-height: 16px !important; max-width: 200px">
+	<span style="line-height: 24px !important;">
+		The toy brought back fond memories of being lost in the rain forest.
+	</span>
+</p>
+```
+
+#### Passed Example 14
+
+This `div` element has three text node descendants. The first one is [empty](#78fd32:empty); the second one is [large enough][] because of the smaller `font-size` on its parent; and the last one is [large enough][] because of the `letter-spacing` specified on its parent. The third `span` element has a [large enough][] text node descendant.
```

Comment:
```suggestion
This `div` element has three text node descendants. The first one is [empty](#78fd32:empty); the second one is [large enough][] because of the smaller `font-size` on its parent; and the last one matches the [cascade](#78fd32:cascade) condition because of the `letter-spacing` specified on its parent. The third `span` element has a [large enough][] text node descendant.
```